proposals/i18n-getOSLanguage.md

```diff
@@ -0,0 +1,78 @@
+# Proposal: i18n.getOSLanguage>
+
+**Summary**
+
+Allow to get the language of your operating system as an `IETF tag`
+
+**Document Metadata**
+
+**Author:** [carlosjeurissen](https://github.com/carlosjeurissen)
+
+**Sponsoring Browser:** TBD
+
+**Created:** 2024-03-18
+
+**Related Issues:**
+https://github.com/w3c/webextensions/issues/252
+https://bugzilla.mozilla.org/show_bug.cgi?id=1888486
+
+## Motivation
+
+### Objective
+
+Allow extensions to display UI in the user's system language, even when this is distinct from the browser's UI language (which is restricted to languages supported by the browser).
+
+#### Use Cases
+
+Having the original OS language is useful for language-related extensions and for extensions who want to provide translations more true to the OS language. For example, to adapt to a specific sub-language, like Belgium-Dutch (nl-BE). While i18n.getUILanguage would return 'nl'.
+
+### Known Consumers
+
+Language-related extensions and extensions wanting to match the language of the OS more closely independent of the langauge of the browser.
+
+## Specification
+
+### Schema
+
+`i18n.getOSLanguage()` would synchronously return an [IETF tag](https://www.w3.org/International/core/langtags/rfc3066bis.html) just like `i18n.getUILanguage()` does right now.
```

Comment:
The proper reference here is IETF BCP47. The current RFCs for that are 5646 and 4647. Generally speaking its common to talk about a "BCP47 language tag" or just "language tag". See also the [I18N Glossary](https://www.w3.org/TR/i18n-glossary) for useful links to I18N terminology for the Web.

Reply:
@aphillips Thanks for the references! Updated the proposal to reflect this!